### PR TITLE
Remove user / application limits on self-hosted

### PR DESCRIPTION
**Pull request by nimish-ks**
## :mag: Overview

This PR make the following critical changes:
- Moved the user / application limiting code outside of the EE folder
- Removed all user / application limits on self-hosted Phase Console

## :bulb: Proposed Changes

- Removed organizational member limits on the free tier of the self-hosted version of Phase Console
- Removed application limits on the free tier of the self-hosted version of Phase Console

Before:
- 3 Applications
- 5 Members

Now:
- ∞ Application
- ∞ Members

## :memo: Release Notes

- Moved the user / application limiting code outside of the EE folder
- Removed all user / application limits on self-hosted Phase Console
- Fixed imports

### :test_tube: Testing

- Run the Phase Console in self-hosted mode and test the free tier application limits 
- Run the Phase Console in self-hosted mode and test the free tier organizational member limits 

### :green_heart: Did You...

- [ ] Ensure linting passes (code style checks)?
- [ ] Update dependencies and lockfiles (if required)
- [x] Verify the app builds locally?
- [ ] Manually test the changes on different browsers/devices?

### CODE REVIEW

**Comment by nimish-ks** [comment]
@rohan-chaturvedi getting `ApolloError: An active invitiation already exists for this user.` on self-hosted in spite of the user being invited the first time.

image

**Comment by rohan-chaturvedi** [comment]
> @rohan-chaturvedi getting `ApolloError: An active invitiation already exists for this user.` on self-hosted in spite of the user being invited the first time.
> 
> image

@nimish-ks  This was a UI issue: there was no loading or disabled state on the "Invite" button, so accidental double clicks were triggering duplicate invites. I've fixed this by adding a loading state to the button, as well as a fix to reset the error message alert when closing the dialog.

**Review by rohan-chaturvedi — CHANGES_REQUESTED** [review]
LGTM, just suggesting a more concise definition of `PLAN_CONFIG`

File: backend/backend/quotas.py [hunk 1]
```diff
@@ -1,21 +1,44 @@
...
+            "max_apps": None,
+            "max_envs_per_app": 10,
+        },
+        "EN": {
+            "name": "Enterprise",
+            "max_users": None,
+            "max_apps": None,
+            "max_envs_per_app": None,
+        },
+    }
```
Comment (rohan-chaturvedi) [inline backend/backend/quotas.py]:
```suggestion
PLAN_CONFIG = {
    "FR": {
        "name": "Free",
        "max_users": 5 if CLOUD_HOSTED else None,
        "max_apps": 3 if CLOUD_HOSTED else None,
        "max_envs_per_app": 3,
    },
    "PR": {
        "name": "Pro",
        "max_users": None,
        "max_apps": None,
        "max_envs_per_app": 10,
    },
    "EN": {
        "name": "Enterprise",
        "max_users": None,
        "max_apps": None,
        "max_envs_per_app": None,
    },
}
```

**Review by rohan-chaturvedi — CHANGES_REQUESTED** [review]

**Review by rohan-chaturvedi — APPROVED** [review]

**Pull request merged**